versioning/scripts/dockerfiles/main.go

```diff
@@ -71,6 +72,17 @@ func deleteIfFileExists(path string) {
 	}
 }
 
+// this function removes all white characters from the string
+// usefull in comparing files that can have a mix of whitespaces and tabs
+func removeWhiteCharacters(str string) string {
```

Comment:
Removing white characters is sub-optimal, as we could potentially ignore real errors like when a space is expected but missing.

Reply:
removeWhiteCharacters doesn't remove spaces - spaces are left. It removes other characters belonging to a category of white characters like tabs, new lines, carriage return.
I agree that leaving spaces is important because they are part of syntax.